VideoApp/VideoApp/ViewControllers/Room/RoomViewModel.swift

```diff
@@ -91,12 +97,21 @@ class RoomViewModel {
         guard let payload = notification.payload as? Room.Update else { return }
         
         switch payload {
-        case .didStartConnecting, .didConnect: delegate?.didConnect()
-        case let .didFailToConnect(error): delegate?.didFailToConnect(error: error)
-        case let .didDisconnect(error): delegate?.didDisconnect(error: error)
-        case .didStartRecording: delegate?.didUpdateRecording()
-        case .didStopRecording: delegate?.didUpdateRecording()
-        case .didAddRemoteParticipants, .didRemoveRemoteParticipants, .didUpdateParticipants: break
+        case .didStartConnecting:
```

Comment:
`VideoApp.Room`...not loving the name...as I revisit the code it should be store or manager or something maybe. I think this is broadcast for sake of being comprehensive or consistent...any change in room state is reported to all observers. But yeah technically `RoomViewModel` could get by without it. Keep in mind `VideoApp.Room` is architected so that multiple objects can easily observe changes...necessary to separate complex logic for `ParticipantsStore` and `MainParticipantStore`...so any state changes are simply sent to everyone.